frontends/parsers/p4/p4parser.ypp

```diff
@@ -955,6 +957,10 @@ typeName
     : prefixedType                     { $$ = new IR::Type_Name(@1, $1); }
     ;
 
+vectorType
```

Comment:
The current PR for the language spec here uses `listType` instead of `vectorType` for the name of this nonterminal: https://github.com/p4lang/p4-spec/pull/1168

Should they be consistent with each other?  If yes, what name for this nonterminal would you recommend?